extensions/htlc/besu-eth/package.json

```diff
@@ -0,0 +1,112 @@
+{
+  "name": "@hyperledger/cactus-plugin-htlc-eth-besu",
+  "version": "0.3.0",
```

Comment:
@AzaharaC Please keep an eye on the PR that is about cutting the 0.4.0 release and if that gets approved/merged before this one does then make sure to update the version here to 0.4.0 as well. If this one goes in first, then no action necessary.

cc: @kikoncuo @takeutak @sfuji822